src/stylus/react-selectize.styl

```diff
@@ -0,0 +1,929 @@
+.example .react-selectize.multi-select .removeable-emoji {
```

Comment:
You should be able to tell Browserify about it so that it gets bundled with the rest of the CSS.